charts/keycloak/values.yaml

```
@@ -155,17 +155,25 @@ livenessProbe: |
   httpGet:
     path: /auth/
     port: http
-  initialDelaySeconds: 300
+  initialDelaySeconds: 5
   timeoutSeconds: 5
 
 # Readiness probe configuration
 readinessProbe: |
   httpGet:
     path: /auth/realms/master
     port: http
-  initialDelaySeconds: 30
+  initialDelaySeconds: 5
```

Reducing this doesn't make much sense. Keep it as it was.
```suggestion
  initialDelaySeconds: 30
```